resources/views/littlelink.blade.php

```diff
@@ -7,7 +7,7 @@
 
 {{-- Fediverse rel="me" links --}}
 @php
-  $relMe = "mastodon, firefish";
+  $relMe = "mastodon, firefish, streams";
```

Comment:
Issue is definitely related to this and possibly the next line. I'm just not sure what the problem is.